X16 Reference - 07 - Machine Language Monitor.md

```diff
@@ -60,5 +62,35 @@ These are the remaining commands:
   * `$` to display the disk directory
   * anything else as a disk command
 
+Monitor Display
+```
+MON
+C*
+   PC  RA RO AC XR YR SP NV#BDIZC
+.;E3bb 01 04 00 00 00 F6 ........
+.
+```
+| Column  | Description |
+|---------|-------------|
+|PC       | Program Counter and the current address
+|RA       | Active RAM Bank
+|RO       | Active ROM Bank
+|AC       | Accumulator Value
+|XR       | X Register Value
+|YR       | Y Register Value
+|SP       | Stack Pointer
+|NV#BDIZC | Status Register
+
+| Status Register Bits | Bit Description |
+|----------------------|-----------------|
+|  N                   | Negative
+|  V                   | Overflow Vector
+| \#                   | Unused
+|  B                   | A Break Point is Active
+|  D                   | Decimal Mode Active (also known as Binary Code Decimal)
+|  I                   | IRQ is Enabled
```

Comment:
I is the interrupt inhibit flag.  If it is enabled, IRQs are masked.

Reply:
Thanks! I updated the text.